react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -525,11 +546,29 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
         @Override
         public void afterTextChanged(Editable s) {
-            if (!mIsSettingTextFromJS && mListeners != null) {
+            if (shouldDelegateTextChangeCalls()) {
                 for (TextWatcher listener : mListeners) {
                     listener.afterTextChanged(s);
                 }
             }
         }
     }
+
+    private boolean shouldDelegateTextChangeCalls() {
+        if (mListeners == null) {
+            // No listeners so, no one to delegate the calls to
+            return false;
+        }
+
+        if (isPreTag()) {
+            // If tag is pre tag we want to delegate calls in every case
+            return true;
+        }
+
+        return !mIsSettingTextFromJS;
+    }
+
+    private boolean isPreTag() {
+        return !TextUtils.isEmpty(mTagName) && mTagName.equals(PRE_TAG);
```

Comment:
This is just a nice-to-have, but I think this could be shortened to be a bit more clear by just reversing the equals check so you get a null and empty check for free since `PRE_TAG` is a constant:

```
return PRE_TAG.equals(mTagName);
```

Reply:
Good point. Fixed.